src/components/Home/Powered.js

```diff
@@ -62,6 +62,20 @@ const Powered = props => (
         </div>
       </Card>
     </CardsContainer>
+    <div>
+      <a
+        className="action-button powered-link centered"
+        href="https://poweredbyaragon.com"
+      >
+        <span>
+          <FormattedMessage
+            id="home.organizations.action"
+            defaultMessage="Learn more"
```

Comment:
Once we have the correct copy for this link, we should update the i18n for this.